dependencies/analyses/README.md

```diff
@@ -46,7 +46,7 @@ Running such an analysis on libpod may look as follows:
 
 ```
 # 1) Compile the binary
-[libpod]$ make podman
+[libpod]$ make bin/podman
```

Comment:
Same here.  Not sure why you need the bin/ here.

Reply:
I answered here https://github.com/containers/libpod/pull/4762#discussion_r361875188